# Microbiome Analysis applied to Vet Med
This repository contains *ad hoc* bioinformatics approaches aimed at microbial population analysis on samples of interest in Veterinary Medicine.

Research group: Elton Vasconcelos, DVM Ph.D; Brian Oakley, Ph.D; Pedro Diniz, DVM Ph.D

College of Veterinary Medicine at Western University of Health Sciences, Pomona, CA

### Our rationale
## Building a Computational Workflow for Microbial Community Analysis in Veterinary Medicine

Metagenomics is a relatively new field that utilizes high-throughput DNA sequencing
technologies to characterize entire communities of microorganisms that are present within
environmental or biological samples of interest. In the last decade, metagenomics has already
had a substantial impact on several Life Sciences disciplines such as evolution, microbial
ecology, human medicine, and public health. Computational algorithms are essential for dealing
with the large volume of data generated by the sequencing assays and, consequently, are capable
of rapidly characterizing microbial taxa (e.g. species) and genes (e.g. antibiotic resistance genes)
present within samples. Important computational advances that have facilitated the adoption of
microbiome research by scientists from a variety of fields include software packages such as ARB,
QIIME, Mothur, Uparse, Dada2, PICRUSt, Anvi'o, among several others. However, each package has pros and cons which eventually force
researchers to develop their own strategies combining tools from different sources that may fit
properly on their data. Our research group has a particular interest on vector-borne zoonotic
diseases in humans and animals. We are currently performing two independent microbial population
studies with the following samples: (i) cat fleas from two different
geographical locations in California (southern CA and northern CA); and (ii)
dog blood samples (either previously diagnosed for tick-borne diseases or reported as
negative). Our customized computational workflow has been deployed as *ad hoc* BASH
scripts that merge several different available tools in order to define the operational taxonomic units
(OTUs) for each sample and accurately classify and compare them. Interesting results have
already been achieved for the flea study and are disclosed in the following publication: 
>**Assessing cat flea microbiomes in northern and southern California by 16S rRNA Next Generation Sequencing.** [Vasconcelos et al., 2018](https://www.ncbi.nlm.nih.gov/pubmed/29893631). *Vector-Borne and Zoonotic Diseases*. 

Ultimately, we have developed an SOP for microbiome analyses applied to canine vector-borne pathogens' investigation:
>**Data analysis workflow for the detection of canine vector-borne pathogens using 16 S rRNA Next-Generation Sequencing.** [Vasconcelos et al., 2021](https://bmcvetres.biomedcentral.com/articles/10.1186/s12917-021-02969-9). *BMC Veterinary Research*.

Please refer to the SOP branch on this github page in order to go through the pipeline in details.

We believe that the approaches we are developing may facilitate
future applications of microbiome research within several veterinary sciences subareas and
fulfill a long-term goal of adoption of metagenomics for routine use by clinicians and/or researchers.
